src/github/nag.rs

```diff
@@ -255,7 +256,7 @@ fn evaluate_nags() -> DashResult<()> {
             };
         }
 
-        if num_active_reviews == 0 && num_active_concerns == 0 {
+        if (num_active_reviews < num_complete_reviews) && num_active_concerns == 0 {
```

Comment:
We should obviously do whatever works best for the core team, but there are other options here than "all reviewed" or "majority reviewed" like 2/3, all but 1 (to account for people being away), etc.